Modules/IO/NIFTI/include/itkNiftiImageIO.h

```diff
@@ -25,6 +25,10 @@
 #include <memory>
 #include "itkImageIOBase.h"
 
+
```

Comment:
Spelling errors in the commit message:
- with not clear -> without clear
- anonomouse -> anonymous